apps/meteor/client/views/room/body/RoomBodyV2.tsx

```diff
@@ -189,11 +190,13 @@ const RoomBody = (): ReactElement => {
 		}
 	`;
 
+	const renderTopic = isPrivateRoom(room) || isPublicRoom(room);
+
 	return (
 		<>
 			<Box position='relative' w='full'>
 				<Box animated className={[wrapperStyle, hideSection && 'animated-hidden'].filter(isTruthy)} ref={sectionWrapperRef}>
-					<RoomTopic room={room} user={user} />
+					{renderTopic && <RoomTopic room={room} user={user} />}
```

Comment:
I think there's a special case for direct rooms inside this component. I think this will break it.